lib/resources/r4/communication.yaml

```diff
@@ -0,0 +1,193 @@
+---
+name: Communication
+field_name_base_url: http://hl7.org/fhir/r4/communication-definitions.html#Communication
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  url: https://hl7.org/fhir/r4/Communication-definitions.html#Communication
+  note: resourceType must be Communication.
+  example: |
+    {
+      "resourceType": "Communication"
+    }
+  action:
+    - create
+
+- name: inResponseTo
+  required: 'no'
+  cardinality: 0..1
+  type: Reference(Communication)
+  description: The Communication this Communication is in response to.
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.inResponseTo
+  note: 
+  example: |
+    {
+      "inResponseTo": {
+        "reference": "Communication/23456_45678",
+        "type": "Communicaiton"
+      }
+    }
+  action:
+    - create
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: The status of the communication
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.status
+  action: create
+  note: As Millenniums Message Center Communicaitons are instantaneously delivered, all communications must be in a "completed" status
+  example: |
+    {
+      "status": "completed""
+    }
+- name: category
+  required: 'yes'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Codeable definition of the Communication Type
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.category
+  note: Message Center Workflows support the categories Message, Reminder, and Coding Queries.  Coding Queries are a Sub-Type of Messages.
+  example: | 
+    {
+      category:
+        {
+          coding: [{
+            "system": "http://terminology.hl7.org/CodeSystem/communication-category",
+            "code": "coding_query"
+        }]
+    }
+  action:
+    - create
+
+- name: priority
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: The priority of the Communication
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.priority
+  action: create
+  note: Millennium only supports Routine and High Prioirities.  Any Priority given other than Routine will be translated to High.  If not provided, the Communication will be treated as Routine.
```

Comment:
Priorities misspelled

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/b51e76543bb04c72ff83076cbc51694bdcb436f1#diff-f08ae1e4c063b9a36a22adaa43fecf21R61

Reply:
priority has a required binding.  Any priority other than Routine will translate to Urgent

Reply:
Since bindings are required by HL7, so we didn't put the binding for priority here.
Update to urgent: https://github.com/cerner/fhir.cerner.com/pull/433/commits/924ad1575295096a798f32771e85ca39f44272c6#diff-f08ae1e4c063b9a36a22adaa43fecf21L64